### DISCUSSION TITLE
echo不能使用，其他都可以

### DISCUSSION THREAD

**Discussion by laiyoi** [nonebot/discussions#406] [问答]
![image](https://github.com/nonebot/discussions/assets/117524251/5eec024e-5bfa-46c7-8ca1-7a711b005509)
![image](https://github.com/nonebot/discussions/assets/117524251/b30fe9a4-c936-466e-b5ac-9db64bf16677)
其他插件的指令都可以响应，只有echo不行

**Comment by yanyongyu** [comment]
你是否配置了command start

**Comment by laiyoi** [comment]
![image](https://github.com/nonebot/discussions/assets/117524251/60a51bf3-4067-4cb3-85a1-b0219f83bace)
配置了

**Reply by yanyongyu** [reply]
echo只有在对机器人私聊或群内at的时候才会回复

**Comment by laiyoi** [comment]
是这样的，知道了
